swift/Readme.md

```diff
@@ -0,0 +1,5 @@
+## Prerequisites for Swift:
+
+    Mac OS X 10.9 (Mavaricks or later)
+
+    XCode 6
```

Comment:
Please mention how to run it? IDE steps/CLI steps

Reply:
Made some changes in Readme.md, please review it.